gitdash/pages/api/github.ts

```diff
@@ -0,0 +1,58 @@
+const { Octokit } = require('@octokit/rest')
+
+export default async (_req: any, res: { status: (arg0: number) => { (): any; new(): any; json: { (arg0: { stars: any; followers: any; starred: any; all_repos: any; repo_count: any; issues: any; issue_count: any }): any; new(): any } } }) => {
+  const octokit = new Octokit({
+    // auth: 
+    // github token for a particular user, leaving empty for now
+  })
+  const username = "prakharrathi25"    // need to get the input from the user for this  
+
+  // Number of followers
+  const followers = await octokit.request(`/users/${username}/followers?per_page=100`)
+  const followerCount = followers.data.length
+
+  // Get all repos 
+  const repos = await octokit.request(`/users/${username}/repos`);
+
+  // Number of stars
+  const starsCount = repos.data.filter((repo: { fork: any }) => !repo.fork).reduce((acc: any, item: { stargazers_count: any }) => {
```

Comment:
Awesome stuff here. `.reduce()` is such a neat function.

Reply:
Thanks a lot!